build.gradle

```diff
@@ -1,5 +1,4 @@
 plugins {
-    id "com.jfrog.bintray" version "1.1"
```

Comment:
By this gradle plugin, error cropped up while building. This plugin isn't used so I removed it.

```
A problem occurred configuring root project 'embulk-output-kintone'.
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not find http-builder-0.7.2.jar (org.codehaus.groovy.modules.http-builder:http-builder:0.7.2).
     Searched in the following locations:
         https://plugins.gradle.org/m2/org/codehaus/groovy/modules/http-builder/http-builder/0.7.2/http-builder-0.7.2.jar

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html
```